### ISSUE TITLE
How to save suggestion history?

### ISSUE THREAD

**Issue by MayankSoni** [Mauker1/MaterialSearchView#132]
Hello. While we click on any suggestion suggestion save successfully. but when we closed application and reopen searchview not showing history what should i do ? please suggest me

**Comment by Mauker1** [comment]
Hello,

Can you elaborate? You're searching for an item that's on the suggestion list, and you want it to show on the search history later on?

**Comment by MayankSoni** [comment]
See. I'm searching apple, banana, carrot. History set in search list. Now it's ok. Now I'm closed application and again open see search list ... Nothing to show previous search item like apple,banana, carrot. Understand. I want when I open app again previous stored list in search item. 
--
Sent from Outlook Email App for Android Wednesday, 02 May 2018, 07:27PM +05:30 from Maurício C. P. Pessoa  notifications@github.com :

**Comment by Mauker1** [comment]
Ok, now I got it. I'll take a look into it to see if I can reproduce this behavior.

**Comment by Mauker1** [comment]
Just for sanity check, can you share the part of your code where you setup the search view?

**Comment by Mauker1** [comment]
Yes, I understand, and I have also tested that. I couldn't reproduce the issue. Can you share your code?

**Comment by MayankSoni** [comment]
Yes I'll share complete code
